docs/docs/tutorial/creating-a-source-plugin/part-1/index.mdx

```diff
@@ -0,0 +1,170 @@
+
+```shell
+git clone https://github.com/gatsbyjs/creating-source-plugin-tutorial.git
+```
+
+SSH snippet:
+
+```shell
+git clone git@github.com:gatsbyjs/creating-source-plugin-tutorial.git
+```
+
+Also be sure to check out [GitHub's documentation for how to clone a repository](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository).
+
+</Collapsible>
+
+After successfully cloning the repository, navigate to its folder and switch to the `start` branch to have a clean boilerplate:
+
+```shell
+git checkout start
+```
+
+Open the repository in your favorite editor as it'll make the next step easier.
+
+## Project structure
+
+The `creating-source-plugin-tutorial` project includes three directories:
+
+- `api`: This is the example backend API
+- `plugin`: The source plugin
+- `site`: An example Gatsby site
+
+So the `site` uses the `plugin` which sources its data from `api`. As mentioned in [Part 0](/docs/tutorial/creating-a-source-plugin/part-0/#development-environment) this project uses yarn workspaces to make this setup work.
+
+By having the plugin and an example site that uses the plugin in the same repository, you can more easily test your plugin.
+
+<Announcement>
+
+You can ignore the `api` folder unless you're curious how a [GraphQL Yoga server](https://github.com/dotansimha/graphql-yoga) is set up. For the purpose of this tutorial the example backend API was included so that this tutorial doesn't rely on outside resources. This way you can run everything locally.
+
+</Announcement>
+
+The `plugin` itself has a structure we'd also recommend following:
+
+- A `package.json` file that sets `gatsby` and `gatsby-plugin` as its `"keywords"` so that Gatsby's plugin search can pick it up
+- A root `gatsby-node.js` that requires `./dist/gatsby-node` (the compiled TypeScript file). At the moment Gatsby looks for a root `gatsby-node.js` file as its entry file.
+- A well-written `README` that explains the user in concise steps how to install, use, and configure your plugin (also see [How to write a plugin README](/contributing/docs-contributions/how-to-write-a-plugin-readme/))
+- A `src` folder with the plugin's source code
+  - A root `gatsby-node.ts` file that exports [Gatsby Node APIs](/docs/reference/config-files/gatsby-node/) from individual files
+  - Individual files that only contain the specific Node API (in [kebab case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case)), e.g. `onPluginInit` becomes `on-plugin-init.ts`. This will also make unit testing your plugin easier.
+
+How you organize your `src` folder is of course up to you, we'd recommend keeping things consistent and files lowercase.
+
+<Collapsible
+  summary={<em>Wondering about a source plugin template for new projects?</em>}
+>
+
+Above we said that you can ignore the `api` folder. This is because for your real-world project you'll most probably source the data from an external API, not an API locally in your repository.
```

Comment:
```suggestion
Above we said that you can ignore the `api` folder. This is because for your real-world project you'll most likely source the data from an external API, not an API locally in your repository.
```

Reply:
+1